<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewMainWindow</class>
 <widget class="QWidget" name="ImageViewMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="AdvancedImageWidget" name="widget" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="loadButton">
        <property name="toolTip">
         <string>Load</string>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/new/prefix1/remove.png</normaloff>:/new/prefix1/remove.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="BitSelectorParametersControlWidget" name="bitSelector">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="DebayerParametersControlWidget" name="parameters" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedImageWidget</class>
   <extends>QWidget</extends>
   <header>uis/advancedImageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BitSelectorParametersControlWidget</class>
   <extends>QFrame</extends>
   <header>filters/ui/bitSelectorParametersControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DebayerParametersControlWidget</class>
   <extends>QWidget</extends>
   <header>corestructs/coreWidgets/debayerParametersControlWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
